torch/_inductor/autotune_process.py

```diff
@@ -1,8 +1,14 @@
+from __future__ import annotations
```

Comment:
I don't think this __future__ import annotations is necessary here, can see if it works without it?

Reply:
Not strictly necessary, but without it we need to use strings for several of the annotations.